python-main.js

```diff
@@ -434,8 +434,17 @@ function web_editor(config) {
         TRANSLATIONS.updateLang(lang, function(translations) {
             config.translate = translations;
             document.getElementsByTagName("HTML")[0].setAttribute("lang", lang);
+            
+            var style = document.createElement('style');
+            style.type = 'text/css';
+            style.innerHTML = '.cssClass { font-weight: bold; color: black; }';
+            document.getElementsByTagName('head')[0].appendChild(style);
+     
+            document.getElementById(lang).className = 'cssClass';
```

Comment:
Right, I see what you meant now.
The way we've done this before is create a css class in the style.css file, and then add it or remove it using jquery.

This is a good example:

https://github.com/bbcmicrobit/PythonEditor/blob/0bc122aec495955525d71b59bcffd4bc831a7b6b/python-main.js#L874-L879